riot/riot.py

```diff
@@ -24,14 +23,24 @@ def __init__(self, *args, **kwargs):
         self.__dict__ = self
 
 
+def rm_singletons(
+    d: t.Dict[str, t.Union[str, t.List[str]]]
+) -> t.Dict[str, t.List[str]]:
+    return {k: [v] if isinstance(v, str) else v for k, v in d.items()}
+
+
+def float_to_list(x: t.Union[float, t.List[float]]) -> t.List[float]:
+    return [x] if isinstance(x, float) else x
```

Comment:
```suggestion
_K = t.TypeVar("_K")
def to_list(x: t.Union[_K, t.List[_K]]) -> t.List[_K]:
    return [x] if not isinstance(x, collections.abc.Iterable) else x
```

and use it in `rm_singletons`

Reply:
Hmm I had tried something like this initially and had gotten:

![image](https://user-images.githubusercontent.com/6321485/100110173-a66bde00-2e3a-11eb-8958-4d75e1903b49.png)

I wasn't using `collections.abc.Iterable` though, so maybe that was it!